opensrp-pnc/src/main/assets/json.form/pnc_registration.json

```diff
@@ -50,1173 +50,2603 @@
+            "err": "Please specify your occupation"
           }
         },
-        "v_required": {
-          "value": "true",
-          "err": "Select at least one outcome for previous pregnancy(s)"
+        {
+          "key": "religion",
+          "openmrs_entity_parent": "",
+          "openmrs_entity": "concept",
+          "openmrs_entity_id": "162929AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+          "openmrs_data_type": "",
+          "type": "native_radio",
+          "label": "Religion",
+          "hint": "Religion",
+          "label_text_style": "bold",
+          "options": [
+            {
+              "key": "christian",
+              "text": "Christian",
+              "openmrs_entity_parent": "",
+              "openmrs_entity": "concept",
+              "openmrs_entity_id": "163125AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+            },
+            {
+              "key": "muslim",
+              "text": "Muslim",
+              "openmrs_entity_parent": "",
+              "openmrs_entity": "concept",
+              "openmrs_entity_id": "162933AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+            },
+            {
+              "key": "traditional",
+              "text": "Traditional",
+              "openmrs_entity_parent": "",
+              "openmrs_entity": "",
+              "openmrs_entity_id": ""
+            },
+            {
+              "key": "pagan",
+              "text": "Pagan",
+              "openmrs_entity_parent": "",
+              "openmrs_entity": "",
+              "openmrs_entity_id": ""
+            },
+            {
+              "key": "none",
+              "text": "None",
+              "openmrs_entity_parent": "",
+              "openmrs_entity": "concept",
+              "openmrs_entity_id": "1107AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+            },
+            {
+              "key": "other",
+              "text": "Other (specify)",
+              "openmrs_entity_parent": "",
+              "openmrs_entity": "concept",
+              "openmrs_entity_id": "5622AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+            }
+          ]
         },
-        "options": [
-          {
-            "key": "alive",
-            "text": "Alive",
-            "openmrs_entity": "concept",
-            "openmrs_entity_id": ""
-          },
-          {
-            "key": "stillbirth_fresh",
-            "text": "Stillbirth-Fresh",
+        {
+          "key": "religion_other",
+          "openmrs_entity_parent": "162929AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+          "openmrs_entity": "concept",
+          "openmrs_entity_id": "160632AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+          "type": "edit_text",
+          "hint": "Write in the woman's other religion that is not on the list.",
+          "edit_type": "name",
+          "relevance": {
+            "step2:religion": {
+              "type": "string",
+              "ex": "equalTo(., \"other\")"
+            }
+          }
+        }
+
+      ]
+    },
+    "step3": {
```

Comment:
remove this step

Reply:
it has been already removed in issue 11. just ignore it. you will not see this in next PR